examples/Cargo.toml

```diff
@@ -20,6 +20,10 @@ webrender = "0.58.0"
 websocket = "0.22"
 ipc-channel = "0.11"
 
+[dependencies.clap]
+version = "2.33"
+default-features = false
```

Comment:
why are you disabling the default features?

I would just add ```clap = "2.33"``` in the dependencies section in alphabetic order.

Reply:
to be honest i was assuming you wouldnt want dependecies bloat and since these features arent critical i can just disable them

default features are suggestions (for typos), color for coloring errors and vecmap for a slight performance, you can see them [here](https://github.com/clap-rs/clap#optional-dependencies--features)

Reply:
I saw the Cargo.lock diff and I got your point.

Though, as Captain Nitpick, I'd still change this to

```
[dependencies]
clap = { version = "2.33", default-features = false }
env_logger = "0.5"
```